slices/xfonts-encodings.yaml

```diff
@@ -0,0 +1,13 @@
+package: xfonts-encodings
+
+essential:
+  - xfonts-utils_copyright
+
+slices:
+  encodings:
+    contents:
+      /usr/share/fonts/X11/encodings/*:
```

Comment:
This will not pick up on `/usr/share/fonts/X11/encodings/large/*`. And it might also be too generic...I'm wondering if there's another package adding files to `/usr/share/fonts/X11/encodings/`? If so, this would generate a conflict later down the road

Reply:
I will look if there is one. In any case I will specify paths more precisely.